pilot/cmd/pilot-agent/status/util/util.go

```diff
@@ -24,11 +24,10 @@ import (
 
 const requestTimeout = time.Second * 1 // Default timeout.
 
-func doHTTPGetWithTimeout(requestURL string, t time.Duration) (*bytes.Buffer, error) {
-	httpClient := &http.Client{
-		Timeout: t,
-	}
+var httpClient = http.Client{}
 
+func doHTTPGetWithTimeout(requestURL string, t time.Duration) (*bytes.Buffer, error) {
+	httpClient.Timeout = t
```

Comment:
I am fairly sure modifying this client is not goroutines safe

Reply:
because re-using the client may be but not modifying the timeout

Reply:
You are right, my mistake.